tasks/check-and-prepare-role-variables.yml

```diff
@@ -0,0 +1,25 @@
+- name: Check cluster configuration variables
+  block:
+    - name: Fail if passwords are not specified
+      fail:
+        msg: "{{ item }} must be specified"
+      when: lookup("vars", item, default="") | length < 1
+      loop:
+        - ha_cluster_hacluster_password
+      run_once: yes
+  when: ha_cluster_cluster_present
+
+- name: Discover cluster node names
+  set_fact:
+    __ha_cluster_node_name: "{{
+        hostvars[inventory_hostname].ha_cluster.node_name
```

Comment:
isn't `hostvars[inventory_hostname].ha_cluster.node_name` the exact same thing as `ha_cluster.node_name`?

Reply:
Oh yes, of course it is. I should have realized that, sorry.